example_systems/2_three_zones_w_electrolyzer/resources/Storage.csv

```diff
@@ -1,7 +1,7 @@
-Resource,Zone,Model,New_Build,Can_Retire,Existing_Cap_MW,Existing_Cap_MWh,Min_Cap_MW,Min_Cap_MWh,Inv_Cost_per_MWyr,Inv_Cost_per_MWhyr,Fixed_OM_Cost_per_MWyr,Fixed_OM_Cost_per_MWhyr,Var_OM_Cost_per_MWh,Var_OM_Cost_per_MWh_In,Self_Disch,Eff_Up,Eff_Down,Min_Duration,Max_Duration,Qualified_Hydrogen_Supply,region,cluster
-MA_battery,1,1,1,0,0,0,0,0,19584,22494,4895,5622,0.15,0.15,0,0.92,0.92,1,10,0,MA,0
-CT_battery,2,1,1,0,0,0,0,0,19584,22494,4895,5622,0.15,0.15,0,0.92,0.92,1,10,0,CT,0
-ME_battery,3,1,1,0,0,0,0,0,19584,22494,4895,5622,0.15,0.15,0,0.92,0.92,1,10,0,ME,0
-MA_battery_for_electrolyzer,1,1,1,1,0,0,0,0,21542,24743,4895,5622,0.15,0.15,0,0.92,0.92,1,10,1,MA,0
```

Comment:
Should put hydrogen batteries in Zone 2 and Zone 3 as well

Reply:
Thank you! I updated the example.